<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>404</width>
    <height>376</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MCSRrank</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="stoppileButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>50</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>停止该桩</string>
    </property>
   </widget>
   <widget class="QLabel" name="showinfo">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>311</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pileamountButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>桩的数量</string>
    </property>
   </widget>
   <widget class="QPushButton" name="showpileButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>某桩状态</string>
    </property>
   </widget>
   <widget class="QPushButton" name="servingcarButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>110</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>服务信息</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pilebillButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>170</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>报表信息</string>
    </property>
   </widget>
   <widget class="QPushButton" name="waitinglistButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>等候情况</string>
    </property>
   </widget>
   <widget class="QLabel" name="modeselectiontext">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>哪个桩</string>
    </property>
   </widget>
   <widget class="QComboBox" name="pileselectioncombo">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>10</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5</string>
     </property>
    </item>
   </widget>
   <zorder>showinfo</zorder>
   <zorder>stoppileButton</zorder>
   <zorder>pileamountButton</zorder>
   <zorder>showpileButton</zorder>
   <zorder>servingcarButton</zorder>
   <zorder>pilebillButton</zorder>
   <zorder>waitinglistButton</zorder>
   <zorder>modeselectiontext</zorder>
   <zorder>pileselectioncombo</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
